docs/www/acls.md

```diff
@@ -275,254 +272,680 @@ The response should look something like this:
 
 Notice that this command uses the Kafka API, so we're using the `--brokers <node IP>:<kafka API port>` pattern here, having the default value as `localhost:9092`.
 
-#### Managing users
+## Authorization
 
-While having a superuser is a must to get started, it's not really a good idea to either share the superuser's credentials everywhere or to make everyone a superuser.
+While **authentication** tells who you are, **authorization** tells you what can you do.
 
-You can create, delete and list users with `rpk acl user`.
+### Access Control Lists (ACLs)
 
-`rpk acl` is an Admin API command so it defaults to connecting to `localhost:9644`. If you have a different IP address, URL, or port, pass `--api-urls <node IP>:<admin API port>`.
+[Access Control Lists](https://en.wikipedia.org/wiki/Access-control_list) (ACLs) is the main mechanism supported by Redpanda to manage user permissions.
+ 
+Usually, other Kafka implementations use Apache ZooKeeper™ to store and manage ACLs. However, since Redpanda doesn't use Zookeeper, we have effectively built our own ACL implementation to store and manage it. You can manage your ACLs with `rpk acl`.
```

Comment:
Zookeeper is only needed for Kafka proper, and that's becoming less true. MSK uses IAM, other implementations use other things, but they aren't necessarily backed by Zookeeper.

I'm not sure explaining other Kafka helps users understand Redpanda. How about merging this paragraph and the next single-sentence paragraph into:

```
Redpanda stores ACLs internally, replicated with Raft to provide the same consensus guarantees as your data. You can manage your ACLs with `rpk acl`.
```
?

Reply:
👍